packages/client/src/components/modal/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode, SyntheticEvent } from 'react'
+import { createPortal } from 'react-dom'
+
+import styles from './styles.module.css'
+
+type Props = {
+  onClose: () => void
+  children: ReactNode
+}
+
+export const Modal = ({ onClose, children }: Props) => {
```

Comment:
_Не критично, но лучше исправить_

Тут можно вынести `children` из `Props`, используя традиционную для функциональных компонентов типизацию через `FC` и `PropsWithChildren`
```suggestion
export const Modal: FC<PropsWithChildren<Props>> = ({ onClose, children }) => {
```

Reply:
Использование `FC` в целом очень хорошая практика, и я советую типизировать все функциональные компоненты с помощью `FC`

[Тут](https://dev.to/elhamnajeebullah/react-typescript-what-is-reactfc-and-why-should-i-use-it-4029) можно почитать про `FC` подробнее